docs/tutorials/01_quadratic_program.ipynb

```diff
@@ -759,39 +759,31 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 16,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "\\ This file has been generated by DOcplex\n",
-      "\\ ENCODING=ISO-8859-1\n",
-      "\\Problem name: CPLEX\n",
-      "\n",
-      "Minimize\n",
-      " obj: _e + 2 f + 3 g\n",
-      "Subject To\n",
-      "\n",
-      "Bounds\n",
-      " 0 <= _e <= 1\n",
-      " 0 <= f <= 1\n",
-      "\n",
-      "Binaries\n",
-      " _e f\n",
-      "End\n",
-      "\n"
+     "ename": "NameError",
```

Comment:
It looks like perhaps just this cell was run - probably best to re-run the whole notebook locally for whats checked in. It will get re-run too when docs are built but it should be checked in a state that would match what CI produces.

Reply:
Thanks - re-ran and checked it.